### PR TITLE
Modify rule S6864: Update examples and removal of LimitRange for now

### PR DESCRIPTION
**Pull request by navendu-barua-sonarsource**
updates in S6864, S6869 and S6870
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6864/kubernetes/rule.adoc [hunk 1]
```diff
@@ -50,9 +50,24 @@ kind: Pod
 metadata:
   name: example
 spec:
-  containers: # Noncompliant
+  containers:
+    - name: web # Noncompliant
+      image: nginx
+----
+or
```
Comment (hendrik-buchwald-sonarsource) [inline rules/S6864/kubernetes/rule.adoc]: I don't think it makes sense to have this additional example, even if we raise an issue for it, as the code just does not make sense.

File: rules/S6864/kubernetes/rule.adoc [hunk 2]
```diff
@@ -75,8 +90,7 @@ spec:
 === How does this work?
 
 A limit can be set through the property `resources.limits.memory` of a
-container. Alternatively, a default limit for a namespace can be set with
```
Comment (hendrik-buchwald-sonarsource) [inline rules/S6864/kubernetes/rule.adoc:78]: Even if we don't detect it yet, we should still keep this sentence as it is a valid way to not be vulnerable.

**Review by hendrik-buchwald-sonarsource — CHANGES_REQUESTED** [review]
Hey, I am not really in favor of most of these changes. Moving the noncompliant comments seems reasonable but some of the other changes I can't comprehend.

**Comment by hendrik-buchwald-sonarsource** [comment]
`validate_asciidoc` is also failing.

File: rules/S6869/kubernetes/rule.adoc [hunk 3]
```diff
@@ -58,28 +55,30 @@ spec:
...
-  limits:
-  - default:
-      cpu: 0.5
-    type: Container
+  containers:
+    - name: web
+      image: nginx
+      resources:
+        limits:
+          cpu: "0.5"
```
[the remark this answers is not in the document]
Comment (navendu-barua-sonarsource) [reply rules/S6869/kubernetes/rule.adoc:75]: While trying to find some more examples [here](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/),  I did not find one that was with floating `resources/limits/cpu`. This is why I changed it to string. Please let me know if the float values are also acceptable

[the remark this answers is not in the document]
Comment (jonas-wielage-sonarsource) [reply rules/S6869/kubernetes/rule.adoc:61]: I would change it back to `0.5` as this is the usual way people define this property.

File: rules/S6870/kubernetes/rule.adoc [hunk 4]
```diff
@@ -39,9 +39,30 @@ kind: Pod
...
-    - name: web
+  containers:
+    - name: web # Noncompliant
       image: nginx
+      volumeMounts:
+        - name: ephemeral
+          mountPath: "/tmp"
+----
+
+[source,yaml,diff-id=2,diff-type=noncompliant]
```
Comment (jonas-wielage-sonarsource) [inline rules/S6870/kubernetes/rule.adoc:50]: The examples in this file with `diff-id=2` do not provide any additional value for me. Please remove them

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM, please see my suggestions before merging.

**Pull request merged**